### PR TITLE
Changelog 2024-02-06

### PR DESCRIPTION
**Pull request by tsg**
Preview: https://xata-cl-pr-28.vercel.app/roadmap#February-6-2024

### CODE REVIEW

File: 2024-02-06.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Improved CSV import error reporting in the UI by showing tooltips above the column names that have errors.
+
+  ![CSV error tooltips](images/csv-error-tooltip.png)
+
+- Small fix to the focus state of the Github connect button on the database cards. Previously the opacity wasn't being switched.
+- Fixed default TTL for the signed URLs created for file attachments when updated via partial update.
+- Fixed some broken links in the OpenAPI spec documentation.
+- For another public website improvement, you can now subscribe to RSS feeds for particular categories of our blog.
+
```
Comment (snide) [inline 2024-02-06.mdx:19]:
```suggestion
- A new example for [uploading files using Next.js](https://github.com/xataio/examples/tree/main/apps/sample-nextjs-upload-file) was added to our examples repository.

```

File: 2024-02-06.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+date: 2-6-2024
+---
+
+### What's new
+
+- We have improved website/docs search experience. As you'd expect, we are using Xata to power the search on our public website, blog and docs. We keep improving the relevancy of the search, and we have now improved the way headings are shown in previews. Please let us know if you have any feedback or if particular search terms don't seem to work as expected.
```
Comment (snide) [inline 2024-02-06.mdx:7]:
```suggestion
- We improved the website search experience, which itself uses Xata's full-text search features. Search results now additionally match against headers within documents, which should make functions and other deeper content easier to find. Please let us know if you have any feedback or if particular search terms don't seem to work as expected.
```

**Review by snide — COMMENTED** [review]
Small ones. Added the new upload sample. I also rewrote the search docs recently, though I don't know if that's worth calling out.

**Comment by tsg** [comment]
Thanks @snide

**Review by alexfrancoeur — APPROVED** [review]
🚢

**Comment by alexfrancoeur** [comment]
Discord:

Hey @everyone, hope you're having a great week so far! Here's this weeks changelog, with a bunch of improvements 🦋 

For a complete list of updates, check out changelogs and upcoming features on our [roadmap](https://xata.io/roadmap).

### What's new

- We improved the website search experience, which itself uses Xata's full-text search features. Search results now additionally match against headers within documents, which should make functions and other deeper content easier to find. Please let us know if you have any feedback or if particular search terms don't seem to work as expected.
- Speaking of search, we have also improved the docs for our search functionality: https://xata.io/docs/sdk/search
- A new example for [uploading files using Next.js](https://github.com/xataio/examples/tree/main/apps/sample-nextjs-upload-file) was added to our examples repository.
- Improved CSV import error reporting in the UI by showing tooltips above the column names that have errors.
- Small fix to the focus state of the Github connect button on the database cards. Previously the opacity wasn't being switched.
- Fixed default TTL for the signed URLs created for file attachments when updated via partial update.
- Fixed some broken links in the OpenAPI spec documentation.
- For another public website improvement, you can now subscribe to RSS feeds for particular categories of our blog

### Community
Congratulations on winning the Holiday Hackathon @cuevantn @christopheanfry @engineereddev Nikhil and Giridhar! 🎉 🎉  Thank you to everyone that participated 🙏 

### Blog posts

- [How to perform Postgres schema changes in production with zero downtime](https://xata.io/blog/zero-downtime-schema-migrations-postgresql) - This contains a recording and summary of the workshop we did about schema changes in Postgres with zero downtime. There's also an in-depth review and demo of [pgroll](https://github.com/xataio/pgroll).
- [Announcing the winners of the Holiday Hackathon](https://xata.io/blog/holiday-hackathon-challenge-winners)

Twitter:

Our most recent changelogs bring some new enhancements to the Xata experience 🦋

⏵ Nested headers returned in the website & docs search results
⏵ A new uploadUrl to simplify client-side uploads for file attachments
⏵ A Xata database integration to Clouldflare workers

Full changelog here: [https://xata.io/roadmap#February-6-2024](https://t.co/Ws1qSQ18ZY)

**Pull request merged**